source/jormungandr/jormungandr/realtime_schedule/tests/cleverage_test.py

```diff
@@ -0,0 +1,322 @@
+# coding=utf-8
+# Copyright (c) 2001-2016, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+# the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+import mock
+from jormungandr.realtime_schedule.cleverage import Cleverage
+import validators
+import datetime
+import pytz
+
+def make_url_test():
+    cleverage = Cleverage(id='tata', timezone='Europe/Paris', service_url='http://bob.com/',
+                          service_args={'a': 'bobette', 'b': '12'})
+
+    url = cleverage._make_url(MockRoutePoint(line_code='line_toto', stop_id='stop_tutu'))
+
+    # it should be a valid url
+    assert validators.url(url)
+
+    assert url.startswith('http://bob.com/')
```

Comment:
for clever age you can even say that the url should be 

`http://bob.com/stop_tutu` no ?